From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: move vcc5v0_usb to the u2phy0 host port

phy-supply = <&vcc5v0_usb> moves from the u2phy1 OTG port to the
u2phy0 host port; label u2phy1_otg. vcc5v0_usb is always-on, so
neither consumer switches it.

Upstream: none; mainline keeps it on u2phy1.

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -780,6 +780,7 @@ &u2phy0 {
 	status = "okay";
 
 	u2phy0_host: host-port {
+		phy-supply = <&vcc5v0_usb>;
 		status = "okay";
 	};
 };
@@ -787,9 +788,8 @@ &u2phy0 {
 &u2phy1 {
 	status = "okay";
 
-	otg-port {
+	u2phy1_otg: otg-port {
 		/* phy for &usbdrd_dwc3_1 */
-		phy-supply = <&vcc5v0_usb>;
 		status = "okay";
 	};
 };
-- 
Armbian

